manifests/install.pp

```diff
@@ -2,9 +2,16 @@
 # @api private
 class katello_devel::install {
   if $facts['os']['release']['major'] == '9' {
-    yumrepo { 'crb':
-      enabled => true,
-      before  => Package['libvirt-devel'],
+    if $facts['os']['distro']['id'] == 'RedHatEnterprise' {
```

Comment:
You can check for the OS **name**, where `RedHat` is RHEL (as opposed to the OS **family** fact)
```suggestion
    if $facts['os']['name'] == 'RedHat' {
```